blazar_dashboard/content/leases/forms.py

```diff
@@ -205,7 +205,8 @@ def handle(self, request, data):
                     'resource_properties': res_props,
                 })
         if data['network_ip_count'] > 0:
-            network_id = conf.floatingip_reservation.get('network_id')
+            network_id = api.client.get_floatingip_network_id(
```

Comment:
I just noticed that we don't have any reasonable error when the operator doesn't define this config value. This PR doesn't make that worse, though. If you have an idea of how to surface such an error to the user/operator nicely, I'm all ears.